texture_manager.c

```diff
@@ -985,13 +870,12 @@ void texture_manager_tick(texture_manager *manager) {
  * textures.  The texture load loop may happen, but it will resolve itself
  * quickly by reloading textures half-sized.
  */
-void texture_manager_memory_warning(texture_manager *manager) {
```

Comment:
It seems like this should continue to take a `texture_manager*` to match the rest of the methods. I think the idea here is not having global state. `m_memory_warning` should be a property on the `texture_manager`.